lib/Twig/Extension/InitRuntimeInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Declares the deprecated Twig_Extension::initRuntime() method.
+ *
+ * Implement explicitely this interface if you really need to implement the
+ * deprecated initRuntime() method in your extensions.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+interface Twig_Extension_InitRuntimeInterface
+{
+    /**
+     * Initializes the runtime environment.
+     *
+     * This is where you can load some file that contains filter functions for instance.
+     *
+     * @param Twig_Environment $environment The current Twig_Environment instance
+     */
+    public function initRuntime(Twig_Environment $environment);
```

Comment:
This interface can be implemented at the same time than Twig_ExtensionInterface only in PHP 5.3.9+, as it defines the same method (remember our issues with the Validator component in Symfony 2.5).
So this makes it unsuited for the Twig 1.x lowest bound (and for usage in Symfony 2.3): https://3v4l.org/PQSTH